README.md

```diff
@@ -111,6 +111,8 @@ versions of Android. If unsupported, the corresponding key is omitted.
 | Key | Value | Notes |
 |-|-|-|
 | `screenOrientation`| portrait, landscape, unknown |
+| `screenWidth` | String | Screen width in density-independent pixels. Beware: this value is different when there is a different screen orientation.  |
```

Comment:
I think all data types are strings, right? But I might remember incorrectly. As documentation, it would be more informative to list how the backend should parse the string. Reading this table does not answer the question of whether it is an int or float